pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -431,7 +435,12 @@ def create_vcf_entry(donor_unique_id, analysis_attrib, gnos_analysis, annotation
     workflow_name = vcf_entry.get('workflow_details').get('variant_workflow_name')
     workflow_version = vcf_entry.get('workflow_details').get('variant_workflow_version')
 
-    if workflow_name == 'SangerPancancerCgpCnIndelSnvStr' and (( workflow_version.startswith('1.0.') or workflow_version.startswith('1.1.'))
+    if workflow_name == 'SangerPancancerCgpCnIndelSnvStr+SVFIX' and (( workflow_version.startswith('1.0.') or workflow_version.startswith('1.1.'))
+            and not workflow_version in ['1.0.0', '1.0.1']):
+        vcf_entry['vcf_workflow_type'] = 'sanger'
+        vcf_entry['vcf_workflow_status'] = 'fixed'
```

Comment:
I am not sure it's the best to use name this new field as *vcf_workflow_status*. As Gordon pointed out in his email (although not directly related to what we do here), what if there is fixed of fixed?

We can follow his idea to use version numbers, eg, *vcf_workflow_result_version*, this time, we populate it as *v2*. If we ever need to produce another fix, it will be another workflow_name and the result_version will be *v3*.

In terms of report, we can generate report for each version when needed.